app/views/layouts/application.html.slim

```diff
@@ -83,14 +83,20 @@ html
                 = yield
 
     javascript:
-      var url = window.location;
-      // Will only work if string in href matches with location
-      $('ul.nav a[href="'+ url +'"]').parent().addClass('active');
 
-      // Will also work for relative and absolute hrefs
-      $('ul.nav a').filter(function() {
-          return this.href == url;
-          }).parent().addClass('active');
+      // Todo: This should be cleaned up
```

Comment:
Ah, that's more of a "who wrote this" than a "who should do this"

That makes sense. Thanks. :smile: